validator/src/validator.rs

```diff
@@ -127,9 +132,13 @@ impl<TNetwork: Network, TValidatorNetwork: ValidatorNetwork>
         let network1 = Arc::clone(&network);
         let (proposal_sender, proposal_receiver) = ProposalBuffer::new();
 
+        // FIXME: Placeholder, update this line :point_down: as soon as the staking contract changes are merged
+        let id = ValidatorId::default(); 
```

Comment:
The staking contract changes are merged, so this should be possible to do correctly now?

Reply:
It is, this branch is outdated (it was updated before the staking contract changes went into _albatross_)